parsl/version.py

```diff
@@ -1,3 +1,3 @@
 """Set module version.
 """
-VERSION = '1.3.0-dev'
+VERSION = '1.3.1-dev'
```

Comment:
don't do this

Reply:
ok, sorry I meant to change that back. I used it to keep track on my machine.